packaging/rpm/centreon-collect.spec

```diff
@@ -1,19 +1,20 @@
 ##
-## Copyright 2021 Centreon
+## Copyright 2023 Centreon
 ##
 
 Summary: Centreon collect's softwares collection
 Name: centreon-collect
 Version: %{VERSION}
 Release: %{RELEASE}%{?dist}
-License:        ASL 2.0
+License: Apache-2.0
 Source: %{name}-%{version}.tar.gz
 Source1: centreonengine_integrate_centreon_engine2centreon.sh
 
 %define thismajor 21.10.0
 %define nextmajor 21.11.0
 
 Group: Applications/Communications
+License: Apache-2.0
```

Comment:
I think only a single "License" entry is allowed withing this section, and within package sections respectively (given what the log says)